app/presenters/chf/file_set_presenter.rb

```diff
@@ -20,6 +20,12 @@ def representative_file_id
       end
     end
 
+    def representative_checksum
+      # I can not explain why this didn't work with a delegate to solr_document
```

Comment:
Cool! Yeah, probably should fix the comment. Otherwise +1 to merge; feel free to go ahead!